src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -52,7 +58,20 @@ internal abstract partial class PatternMatcher : IDisposable
             bool allowFuzzyMatching = false)
         {
             culture = culture ?? CultureInfo.CurrentCulture;
+            if (culture.Name == "en-US")
```

Comment:
> (Mind you, VS sets current culture to match its active language pack, so you may be able to rely on en-US here based on that, but I don't know which other processes might invoke this code.)

Actually, this might not even be true. It sets the CurrentUICulture, but possibly leaves CurrentCulture alone. I'd have to check, but I still don't think this is the right approach, so it's kind of beside the point.